src/hdf5/osw_2ioda.py

```diff
@@ -298,7 +303,7 @@ def adjust_dateTime(obs_DF, dat_ref):
 
 
 def adjust_longitude(obs_DF, osw_source):
-    if osw_source in ('CYGNSS', 'Spire'):
+    if osw_source in ('CYGNSS', 'Spire', 'Spire-L2'):
         mask = obs_DF['longitude'] > 180
         obs_DF.loc[mask, 'longitude'] = obs_DF['longitude'].loc[mask].values - 360
```

Comment:
```suggestion
    obs_DF.loc[obs_DF['longitude'] > 180, 'longitude'] -= 360
```

Reply:
and thinking the section doesn't even need a conditional check

basically if *any* longitude provided is greater than 180,, then subtract by 360